src/xtgeo/surface/_regsurf_cube_window.py

```diff
@@ -4,9 +4,11 @@
 
 import numpy as np
 import numpy.ma as ma
-from xtgeo.common import XTGeoDialog
-from xtgeo.common import XTGShowProgress
-from . import _regsurf_cube_window_v2 as cwv
+
+from xtgeo.common import XTGeoDialog, XTGShowProgress
+
+from . import _regsurf_cube_window_v2 as cwv2
+from . import _regsurf_cube_window_v3 as cwv3
```

Comment:
I think you could skip the `from` and just `import ._regsurf_cube_window_v2 as cwv2`

Reply:
![image](https://github.com/equinor/xtgeo/assets/16700455/5e9e3bcf-09ae-420d-b867-008a78af7a07)